high-level-tests/e2e/cypress/integration/campaign-management.feature

```diff
@@ -23,11 +23,21 @@ Fonctionnalité: Gestion des Campagnes
     Et je clique sur "Résultats collectifs"
     Alors je vois la moyenne des résultats à 50%
 
-  Scénario: Je créé une campagne
+  Scénario: Je créé une campagne d'évaluation
```

Comment:
Hum est-ce que c'est pas doublon avec les tests d'acceptance au final ?

Reply:
On rentre pas autant dans le détail ici, non ?

Reply:
ouais c'est plus par rapport au tests end to end je maitrise pas trop le truc mais avoir un seul cas ou on crée une campagne (peu importe le type) comme test end to end ça suffit pas ? C'est assez long  à exécuter comme test du coup  je sais pas si on doit vraiment tout tester avec ces tests. C'est plus une question qu'un retour

Reply:
C'est des cas nominaux que vont tester les POs en recette donc on les automatisé. Par contre on ne fait pas de tests e2e sur des cas à la marge à mon avis 😄